src/components/CardComponent.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+    <Col :xs="20" :sm="16" :md="12" :lg="8" :xl="6">
+        <div id = "cardComponent" align="center">
+            <MovieDetailsComponent v-if = "isMovieCard" :movie="movie" />
+            <PersonDetailsComponent v-if = "!isMovieCard" :person = "isActor ? actor : producer" :isActor="isActor" />
```

Comment:
- Please rename component names as it is known by default it is component , component names can be changed from MovieDetailsComponent --> MovieDetails or Movie, PersonDetailsComponent --> PersonDetails or Person